src/lib/config.js

```diff
@@ -39,8 +39,7 @@ const fs = require('fs')
 class Config {
   getFileContent (path) {
     if (!fs.existsSync(path)) {
-      console.log(`File ${path} doesn't exist, can't enable JWS signing`)
```

Comment:
Why remove this line? I asked for this to be added so we do not depend on the consumer of this class to log this error condition.

Reply:
One reason is because I thought it is redundant since the error bellow logs the same thing, but also the reason that I noticed it is because `console.log` breaks linting rules.

Reply:
@bushjames I can revert that if you want and suppress the eslint error for now.

Reply:
well, as a general rule I prefer to write a log entry before throwing an exception as a mitigation against the exception error message being obscured by the catcher. there is no logger in context here so console is only option.